Concentus.Oggfile/OpusOggReadStream.cs

```diff
@@ -70,11 +71,31 @@ public OpusOggReadStream(OpusDecoder decoder, Stream stream)
         /// </summary>
         public long PageGranulePosition { get; private set; }
 
+        /// <summary>
+        /// Gets the sample position in the stream.
+        /// </summary>
+        public long Position => PageGranulePosition * _decoder.NumChannels * BytesPerSample;
```

Comment:
1) What are you trying to expose here and
2) Whatever it is I'm pretty sure it's calculated wrong - see also my other comments about how "sample position" is arbitrary depending on your decoder configuration